stage0/src/acpi.rs

```diff
@@ -373,12 +494,15 @@ impl RomfileCommand {
             Some(CommandTag::WritePointer) => {
                 Ok(Command::WritePointer(unsafe { &self.body.wr_pointer }))
             }
+            Some(CommandTag::AddPciHoles) => {
+                Ok(Command::AddPciHoles(unsafe { &self.body.pci_holes }))
+            }
             _ => Err("Invalid command tag in table-loader"),
         }
     }
 
     fn invoke(&self, fwcfg: &mut FwCfg, acpi_digest: &mut Sha256) -> Result<(), &'static str> {
-        if self.tag > 0x80000000 {
+        if self.tag > 0x80000001 {
```

Comment:
Can you refer to the constant above?

Reply:
Is there a neat way to request the max value from an enum?

Reply:
Not that I know of, but you can define a `const MAX_VALUE = AddPciHoles` at the top and use that, to make things even clearer.

Reply:
I've changed it a bit more; the idea was that if we encounter a unknown command in the lower 31 bits, we error out and panic, whereas when we encounter an unknown vmm-specific command (in the higher 31 bits) we just log an error and continue.

That logic should be a bit more explicit now.